DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: deparse
 Title: What the Package Does (one line, title case)
 Version: 0.0-0
-Authors@R: person("Kirill", "Müller", role = c("aut", "cre"), email = "krlmlr+r@mailbox.org")
+Authors@R: person("Kirill", "M�ller", role = c("aut", "cre"), email = "krlmlr+r@mailbox.org")
```

Comment:
The multibyte character was introduced here. DESCRIPTION should be in UTF-8, as stated in the `Encoding` field.